src/id/mangkomik/src/eu/kanade/tachiyomi/extension/id/mangkomik/SirenKomik.kt

```diff
@@ -27,6 +27,8 @@ class SirenKomik : MangaThemesia(
 
     override fun chapterListSelector() = ".list-chapter a"
 
+    override val pageSelector: String = "Deactivate html search"
```

Comment:
Are you using this because it will never match, or because this is actually the elements on the page?

If the former, you should make it more obvious, and use a selector that will always fail (along with a comment):
```css
> :not(*)
```